.github/workflows/autocorrect-lint.yml

```
@@ -0,0 +1,16 @@
+name: AutoCorrect Lint
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: AutoCorrect
+        uses: huacnlee/autocorrect-action@main
```

We tend to avoid using `main` and instead opt for a version or tag such as `v1.5.5`. I believe you can change the above to:

```suggestion
        uses: huacnlee/autocorrect-action@v1.5.5
```